src/main/target/BETAFLIGHTF3/target.h

```diff
@@ -28,8 +28,8 @@
 //#undef USE_GYRO_OVERFLOW_CHECK
 //#undef USE_GYRO_LPF2
 
-#undef USE_ITERM_RELAX
```

Comment:
In my case I have few old quads that I used for testing. I personally think flight stuff is prefered above led strip, but let's see at the end how it will go and what can be optimized

Reply:
So yeah I am fine with it for now

Reply:
Should this go into 3.5 as well (if so, it will most likely also need an explicit `#undef USE_LED_STRIP` to fit into flash)?

Reply:
No leave it for 3.5. That would remove some features for users.
for 4.0 it is good to test with it and maybe at the end fit it